### PR TITLE
Fix requirements for correct testing

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

I am targeting this branch, because it is BC.

This PR fix tests and change some minimum requirements.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "sonata-project/comment-bundle": "^3.1",
         "sonata-project/core-bundle": "^3.13.0",
         "sonata-project/datagrid-bundle": "^2.3",
+        "sonata-project/doctrine-extensions": "<=1.3.1",
```
Comment (core23) [inline composer.json]:
```suggestion
        "sonata-project/doctrine-extensions": "^1.3.1",
```

Comment (wbloszyk) [reply composer.json]: @core23 but we need 1.0 - 1.3.1

Comment (wbloszyk) [reply composer.json]: anyway it is sonata-project/doctrine-extensions issue
Sonata\Doctrine\Test\EntityManagerMockFactory::create try use $test->createMock() but it is protected method.

Comment (core23) [reply composer.json]: You could use the fresh 1.5.1 release of the doctrine-extensions 🚀

Comment (wbloszyk) [reply composer.json]: But its make errors

Comment (core23) [reply composer.json]: > But its make errors

That's right, but you can fix the failing test easily

**Comment by greg0ire** [comment]
I think bumping the phpunit-bridge to 5.0 might fix the build

**Comment by greg0ire** [comment]
It worked, congrats!

File: .travis/install_test.sh [hunk 2]
```diff
@@ -11,3 +11,5 @@ wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coverall
 chmod u+x "${HOME}/bin/coveralls"
 
 composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
+composer show --latest 'sonata-project/*'
+composer show --latest 'symfony/*'
```
Comment (greg0ire) [inline .travis/install_test.sh]: This should be removed, it was for debugging purposes I supposed? dev-kit will remove it if you don't

**Comment by greg0ire** [comment]
> Test should be updated!!!

What do you mean exactly?

**Comment by wbloszyk** [comment]
@greg0ire 
I just do it. EntityManagerMockFactory::create(TestCase....) try call protected method TestCase->createMock. Also EntityManagerMockFactory is deprecated so i update test by EntityManagerMockFactoryTrait

**Comment by wbloszyk** [comment]
@greg0ire 
Sorry, i miss one changelist :( 
Now should be OK

**Comment by greg0ire** [comment]
Thanks @wbloszyk !

**Pull request merged**